gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -200,6 +200,9 @@ const CssVariablesPage = () => {
             where <NxCode>box-shadow</NxCode> isn't supported, such as SVG elements, use this instead
             as the value of a <NxCode>filter</NxCode> <NxCode>drop-shadow</NxCode>.
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-box-shadow-dropdown">
```

Comment:
These variables are grouped based on what CSS property they are for, so this one should be up with `--nx-box-shadow-focus`

Reply:
Updated here c88f1abdb10667054bc42ea3d8ef210e6d81eaf1.